data/map planets.txt

```diff
@@ -4958,3 +4958,67 @@ planet Zydee
 	landscape land/enceladus_1
 	description `Although this moon initially seems barren and lifeless, there is evidence that the nearby Gegno Scin have recently attempted to set up an outpost here. Several partially-constructed buildings and landing zones are scattered about the surface, but none of them show any signs of activity, likely due to recent warfare with the Gegno Vi. Despite this, a small number of satellites actively orbit Zydee, likely providing important tactical information such as the volume of system traffic.`
 	government Uninhabited
+
+
+# Pirate Stuff
+
+planet Gagarin
+	attributes CCOR factory urban
+	landscape land/city0
+#	description "Gagarin is the oldest-settled planet in the CCOR and the most populated. All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... though the grid hasn't been able to keep pace with demand from a growing population. Although they were designed to be expanded with demand over time, such resource-intensive infrastructure projects became harder and harder to supply the materials for."
+#	spaceport "The heart of the planetary infrastructure network the spaceport is perpetually crowded. Materials are constantly coming and going with ships landing, taking off, or being moved to underground hangars if they can't be unloaded quickly. The spaceport,at least, appears as if, albeit just barely, it's able to meet demand"
+	outfitter "Ammo South"
+	outfitter "Basic Outfits"
+	outfitter "Common Outfits"
+	outfitter "Pirate Outfits"
+	bribe 1
+	security 0.33
+
+planet Bailey
+	attributes CCOR mining
+	landscape land/lava3
+#	description "Bailey is a dead, nearly tidally locked, world scarcely larger or more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Though miners must regularly be on the move it has brought several mining expeditions to the planet. Like aphids sucking the sap from plants, semi-mobile mining stations strip and refine the minerals so that they may be used by the CCOR."
+	spaceport "Given the roving nature of the settlements, and the lack of general atmosphere, the spaceport and other major infrastructure are located in orbit. Transit to and from the planet is facilitated by a clever network of skyhooks, which utilize smaller amounts of rare and expensive resources compared to the more common repulsor technology used elsewhere."
+	bribe 1
+	security 0.33
+
+planet "M.B. Station"
+	attributes CCOR military station urban
+	landscape land/city6
+	description "The hub of military logistics for the CCOR Mordente-Bridi station is as close to a fortress as can be built without the benefit of an asteroid field. The twin cylinders that make up the outer hull are layered with the strongest armor CCOR can manufacture at scale. The Bridi Cylinder is pressurized containing crew quarters and logistical facilities, and the Mordente contains CCOR's shipyard."
```

Comment:
```suggestion
	description "The hub of military logistics for the CCOR, Mordente-Bridi station is as close to a fortress as can be built without the benefit of an asteroid field. The twin cylinders that make up the outer hull are layered with the strongest armor CCOR can manufacture at scale. The Bridi Cylinder is pressurized containing crew quarters and logistical facilities, while the Mordente Cylinder contains CCOR's main shipyard."
```

Reply:
Rewrote M.B. Station a little for some necessary changes.